v1/providers/nebius/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore all Markdown
+*.md
```

Comment:
why? also, was `NEBIUS_TESTING_GUIDE.md` supposed to make it passed this ignore?

Reply:
Good call out. Honestly, none of those Markdown files excluding the README should be preserved. Let me clean this up and check in changes.